### PR TITLE
Cmr 517

### PR DESCRIPTION
**Pull request by dseelig**

### CODE REVIEW

File: src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java [hunk 1]
```diff
@@ -32,10 +33,19 @@ public void contextDestroyed(ServletContextEvent arg0) {
...
-        DissTerms dissTerms = new DissTerms(context.getInitParameter("config.path"));
-        context.setAttribute("dissConf", dissTerms);
 
-        SetsConfig sets = new SetsConfig(context.getInitParameter("config.path"));
-        context.setAttribute("sets", sets);
+        try {
+            DissTermsDao dissTerms = new DissTermsDao(context.getInitParameter("config.path"));
+            context.setAttribute("dissConf", dissTerms);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```
Comment (claussni) [inline src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java]: printStrackTrace()!

File: src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java [hunk 2]
```diff
@@ -32,10 +33,19 @@ public void contextDestroyed(ServletContextEvent arg0) {
...
+        }
+
+        try {
+            SetConfigDao setConfig = new SetConfigDao(context.getInitParameter("config.path"));
+            context.setAttribute("sets", setConfig);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
```
Comment (claussni) [inline src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java]: printStrackTrace()!

File: src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java (hunk 1 above)
Comment (dseelig) [reply src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java]: Replace with error log in catch block.

File: src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java (hunk 2 above)
Comment (dseelig) [reply src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java]: Replace with error log in catch block.

File: src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java [hunk 3]
```diff
@@ -38,14 +43,14 @@ public void contextInitialized(ServletContextEvent sc) {
             DissTermsDao dissTerms = new DissTermsDao(context.getInitParameter("config.path"));
             context.setAttribute("dissConf", dissTerms);
         } catch (FileNotFoundException e) {
-            e.printStackTrace();
+            logger.error(context.getInitParameter("config.path") + " not found.", e);
```
Comment (claussni) [inline src/main/java/de/qucosa/oai/provider/application/ApplicationConfigListener.java]: Application cannot function without the configuration. Better to terminate the context initialization then with a RuntimeException.

**Pull request merged**